noetic/distribution.yaml

```diff
@@ -1550,6 +1550,17 @@ repositories:
       url: https://github.com/stack-of-tasks/eiquadprog.git
       version: devel
     status: maintained
+  ergodic_exploration:
+    doc:
+      type: git
+      url: https://github.com/bostoncleek/ergodic_exploration.git
+      version: v1.0.0
+    release:
+      tags:
+        release: release/noetic/{package}/{version}
+      url: https://github.com/bostoncleek/ergodic_exploration-release.git
+      version: 1.0.0-1
```

Comment:
Please add a `source` entry and a top-level LICENSE file as-per our [review guidelines](https://github.com/ros/rosdistro/blob/master/REVIEW_GUIDELINES.md#types-of-pull-requests). Thanks!

Reply:
I will add those. Is there a way to update the release repo using bloom? Or do I need to close this PR and open a new? This is the first time this package has been released.

Reply:
To update this PR with a `source` entry, you can manually modify and push to the rosdistro branch created by bloom: https://github.com/bostoncleek/rosdistro/tree/bloom-ergodic_exploration-0

You shouldn't need to update the release repo located here: https://github.com/bostoncleek/ergodic_exploration-release.git

Reply:
Alternatively, I think you can rerun bloom-release with the `--edit` option to update information and let it open a fresh PR.

Reply:
I was under the impression I needed to update the release repo because the license file was added. Is this not the case? Either way I had to move the release tag to a different commit and this would need to be reflected in the release repo.

Reply:
I think as long as the LICENSE file is in the source repository, we're happy. Sorry for any confusion.

Reply:
No worries. I had to make another modification in the source repo that should be in the release rep too. So I will close this PR and open a fresh one using bloom.